lib/integrations/api_exceptions.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Integrations
+  module ApiExceptions
+    APIExceptionError = Class.new(StandardError)
+    BadRequestError = Class.new(APIExceptionError)
+    UnauthorizedError = Class.new(APIExceptionError)
+    ForbiddenError = Class.new(APIExceptionError)
+    # ApiRequestsQuotaReachedError = Class.new(APIExceptionError)
```

Comment:
I'm assuming a couple of these Exceptions commented out because v1 of the spec hasn't defined them but will be available in a upcoming spec?

Reply:
I included those but commented them out because they are common exceptions for integration's/api's but the ga4gh api doesn't use them, or doesn't use them yet.
The main idea of this file is that any integration we add uses the exception subclasses of `APIExceptionError` from this module instead of forcing the business logic to figure out which REST exceptions or other api type exceptions could be thrown.

Reply:
Thanks for the explanation!